### PR TITLE
Added Wazuh Cloud info and updated nav instructions.

### PR DESCRIPTION
**Pull request by janettelynch**
# Description

Thank you for contributing to a JupiterOne integration!

Please include a summary of the change and which issue is fixed. Please also
include relevant motivation and context. List any dependencies that are required
for this change.

## Type of change

Please leave any irrelevant options unchecked.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### General Development Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### Integration Development Checklist:

Please leave any irrelevant options unchecked.

- [ ] I have checked for additional permissions required to call any new API
      endpoints, and have documented any additional permissions in
      `jupiterone.md`, where necessary.
- [ ] My changes properly paginate the target service provider's API
- [ ] My changes properly handle rate limiting of the target service provider's
      API
- [ ] My new integration step is instrumented to execute in the correct order
      using `dependsOn`
- [ ] I have referred to the
      [JupiterOne data model](https://github.com/JupiterOne/data-model/tree/main/src/schemas)
      to ensure that any new entities/relationships, and relevant properties,
      match the recommended model for this class of data
- [ ] I have updated the `CHANGELOG.md` file to describe my changes
- [ ] When changes include modifications to existing graph data ingestion, I've
      reviewed all existing managed questions referencing the entities,
      relationships, and their property names, to ensure those questions still
      function with my changes.

### CODE REVIEW

File: docs/jupiterone.md [hunk 1]
```diff
@@ -35,9 +35,22 @@ connects directly to Wazah Manager APIs to obtain agent information. Customers
 authorize access to their self-hosted servers by providing the manager base URL
 and a username and password to JupiterOne.
 
+### In Wazuh Cloud
+
+If you are using Wazuh Cloud, contact Wazuh Support to make changes to your Wazuh 
+Cloud account to expose the API. In addition, provide Wazuh Support the following
+J1 IP addresses that you must add to the allowlist:
+- 18.219.33.157
```
Comment (mdaum) [inline docs/jupiterone.md:43]: let's get rid of these IPs. We have a diff set for each production env, and we will just have customers reach out to us for the right set to whitelist.

File: docs/jupiterone.md [hunk 2]
```diff
@@ -35,9 +35,22 @@ connects directly to Wazah Manager APIs to obtain agent information. Customers
 authorize access to their self-hosted servers by providing the manager base URL
 and a username and password to JupiterOne.
 
+### In Wazuh Cloud
+
+If you are using Wazuh Cloud, contact Wazuh Support to make changes to your Wazuh
```
Comment (mdaum) [inline docs/jupiterone.md:40]: Let's change this to:

> If you are using Wazuh Cloud, please contact J1 support, as we will need to provide you a list of IPs for Wazuh to whitelist. After getting that list from us, you will need to contact Wazuh Support to make changes to your Wazuh....(the rest of what you already have).

File: docs/jupiterone.md [hunk 3]
```diff
@@ -35,9 +35,22 @@ connects directly to Wazah Manager APIs to obtain agent information. Customers
...
+If you are using Wazuh Cloud, contact Wazuh Support to make changes to your Wazuh 
+Cloud account to expose the API. In addition, provide Wazuh Support the following
+J1 IP addresses that you must add to the allowlist:
+- 18.219.33.157
+- 18.218.86.86
+- 52.14.136.234
+
+After you have done this, the J1 integration can connect to your Wazuh Cloud instance. 
+Be sure to structure your MANAGER_URL in the J1 integration configuration to be in the 
+following format: https://${wauzhCloudEnvId}.cloud.wazuh.com/api/wazuh.
```
Comment (mdaum) [inline docs/jupiterone.md:49]: let's put back ticks around the template uri:

`https://${wauzhCloudEnvId}.cloud.wazuh.com/api/wazuh`
and drop the period at the end, as many will copy/paste this.

**Review by mdaum — CHANGES_REQUESTED** [review]

**Comment by janettelynch** [comment]
@mdaum Could you review and approve? Thanks!

**Review by VDubber — DISMISSED** [review]

**Review by mdaum — APPROVED** [review]

**Pull request merged**